test/utils/config.go

```diff
@@ -198,3 +204,35 @@ func WaitStartupMsg(t *testing.T, snap, expectedMsg string, since time.Time, ret
 	}
 	t.Fatalf(`Time out: reached max %d retries looking for "%s"`, retries, expectedMsg)
 }
+
+// DisableConfigProviderPlatformSnap disables the config provider for the specified app
+// and sets the common configuration path
+func DisableConfigProviderPlatformSnap(t *testing.T, snap, app string) {
+	SnapSet(t, snap, "apps."+app+".config.edgex-config-provider", "none")
+	SnapSet(t, snap, "apps."+app+".config.edgex-common-config", "./config/core-common-config-bootstrapper/res/configuration.yaml")
+}
+
+// DisableConfigProviderServiceSnap disables the config provider for the specified app,
+// copies the common configuration file from the platform snap to the service snap,
+// and sets the common configuration path.
+func DisableConfigProviderServiceSnap(t *testing.T, snap, app string) {
+	SnapSet(t, snap, "apps."+app+".config.edgex-config-provider", "none")
+
+	t.Logf("Copying coommon config file from platform snap to service snap: %s", snap)
+
+	sourceFile := "/snap/edegxfoundry/current/config/core-common-config-bootstrapper/res/configuration.yaml"
```

Comment:
This won't work unless we run the Go test with `sudo`. Otherwise, we won't have permission to create a file under `/var/snap`.

Reply:
You could instead copy by executing the `sudo cp` command.